content/2.application.md

```diff
@@ -42,10 +42,13 @@ The Participant node is a node in the Corda business network with the Payments C
 ### Payments CorDapp
 
 The Payments CorDapp is a custom CorDapp developed by the Participant or the BNO using the Gevamu Payments SDK. 
+In order to connect the CorDapp with the Gevamu gateway associated with the PSP, the developer must include details of the PSP.
+Therefore, with minimal changes to the CorDapp, it can work with different PSPs.
 
 ### Gateway node
 
 The Gateway node is a node in the Corda business network where the Gevamu Payments Gateway is installed as a CorDapp. The Gateway node communicates on-chain with the Participant’s CorDapp based on the Gevamu Payments SDK via a proprietary API; the connection with the PSP is off-chain.  
+Each Gevamu gateway is custom written to work with a specific PSP.
```

Comment:
```suggestion
Each Gevamu Payments Gateway is custom-written to work with a specific PSP.
```

Reply:
@Hdd87 I'd move this sentence closer to the beginning of the paragraph - let's put it after the first sentence.